lib/vector-core/src/event/metric.rs

```diff
@@ -265,22 +264,26 @@ impl Metric {
         self
     }
 
+    #[must_use]
```

Comment:
Similarly, I don't think ignoring the return value of these functions is likely to cause issues. Especially since they mutate `self`. If they returned a copy, I could maybe see using `must_use` to ensure users store the updated value.

Reply:
Removed in a4bf98e